lib/u3d/unity_version_number.rb

```diff
@@ -43,7 +43,7 @@ def to_s
   class UnityVersionComparator
     include Comparable
 
-    RELEASE_LETTER_STRENGTH = { a: 1, b: 2, f: 3, p: 4 }.freeze
+    RELEASE_LETTER_STRENGTH = { a: 1, b: 2, f: 3, p: 4, xa: 1, xb: 2, xf: 3, xp: 4 }.freeze
```

Comment:
I would prefer a separate fix for the Linux fix, in particular with unit tests for the elements we have tests for. E.g. the UnityVersionComparator, etc.